hdf5-vol-daos.spec

```diff
@@ -1,11 +1,11 @@
 %global with_mpich 1
 %global with_openmpi3 1
 
-%global vol_test_tag 0.9.0
+%global vol_test_tag 0.9.1
```

Comment:
I would like to understand why we use the tag here though since the test vol version is specified in the daos-vol submodule.

Reply:
We are pulling is the test repo separately and not doing a git submodule update
The spec file will do the following with the test repo
%setup -n vol-daos-%{vol_tag}
%setup -T -D -b 1 -n vol-daos-%{vol_tag}
mv ../vol-tests-%{vol_test_tag}/* test/vol/

From the build log this translates to:
cd /builddir/build/BUILD
+ rm -rf vol-daos-1.1.0rc2
+ /usr/bin/gzip -dc /builddir/build/SOURCES/v1.1.0rc2.tar.gz
+ /usr/bin/tar -xvvf -

cd vol-daos-1.1.0rc2
+ /usr/bin/chmod -Rf a+rX,u+w,g-w,o-w .
+ cd /builddir/build/BUILD
+ /usr/bin/gzip -dc /builddir/build/SOURCES/v0.9.1.tar.gz
+ /usr/bin/tar -xvvf -

+ cd vol-daos-1.1.0rc2
+ /usr/bin/chmod -Rf a+rX,u+w,g-w,o-w .
+ mv ../vol-tests-0.9.1/* test/vol/
+ exit 0

Reply:
> I would like to understand why we use the tag here though since the test vol version is specified in the daos-vol submodule.

@mchaarawi Packages (both RPMs and DEBs) are built from upstream release packages, typically tarballs.  That is inherent in the design of the packaging tools made by the distros.

If upstream is going to maintain and release vol_daos and vol_test as separate projects, that dictates that we need to consume those releases separately.

Reply:
i see. thx for the explanation.